src/traits/election/vrf.rs

```diff
@@ -262,13 +248,18 @@ where
         })
     }
 
-    fn generated_from_seed_indexed(_seed: [u8; 32], index: u64) -> (Self, Self::PrivateKey) {
+    // Sishan NOTE TODO: this one should be substitude by the one in bn254_priv.rs or ed25519_priv.rs or balabala, use SIGSCHEME;
+    // Maybe the trait SignatureScheme should be deleted...At least for BLSoverBN254CurveSignatureScheme, the function inside is useless.
+    fn generated_from_seed_indexed(seed: [u8; 32], index: u64) -> (Self, Self::PrivateKey) {
```

Comment:
+1, Let's make an issue to do this in the future!

Reply:
Just made one https://github.com/EspressoSystems/HotShot/issues/1512